src/main/resources/assets/not_enough_gamerules/lang/en_us.json

```diff
@@ -6,6 +6,9 @@
   "gamerule.anvilDamage": "Deal anvil damage",
   "gamerule.canCopperOxide": "Can copper oxide",
   "gamerule.canCopperOxide.description": "Defines if normal copper can oxidize",
+  "gamerule.canEntityInteractWithBlock": "Can Entity Interact With Blocks",
+  "gamerule.canEntityInteractWithEntity": "Can Entity Interact With Another Entities",
+  "gamerule.canEntityInteractWithWorld": "Can Entity Interact With World",
```

Comment:
No need to make every work start with a cap
`Can entity interact with block` is good enough...